runtime/doc/message.txt

```diff
@@ -315,10 +315,9 @@ generate this message.  This could be caused by a |-m| commandline argument.
 You can switch the 'write' option on with ":set write".
 
 							*E25*  >
-  GUI cannot be used: Not enabled at compile time
+  Nvim does not have a built-in GUI
```

Comment:
The message here should be exactly the same as the actual error message. Either 'have a built-in GUI' or 'have built-in GUI', but used consistently (I prefer the former).